openedx/api.py

```diff
@@ -515,6 +515,40 @@ def get_edx_api_service_client():
     return edx_client
 
 
+def get_edx_retirement_service_client():
```

Comment:
Can we write a test for this? Something similar to `test_get_edx_api_client` in `openedx/api_test.py`.